### ISSUE TITLE
Apply facial effects?

### ISSUE THREAD

**Issue by tjasmin111** [sicxu/Deep3DFaceRecon_pytorch#160]
Very interesting work. I am wondering how can we use it for facial effects like making nose smaller or eyes bigger? Is these a straight forward way to do it that can be done in real time?

Any hints would be helpful

**Comment by Siddharth1698** [comment]
Hey @tjasmin111  ,
Wondering if you were able to do some manipulation for facial effects.

**Comment by tjasmin111** [comment]
Nothing yet. I still don't know how those beauty apps do this in realtime. Do they make a 3D model of the face?!

**Comment by Siddharth1698** [comment]
Not exactly sure, they maybe using lightweight generative models too. If we could manipulate code in bfm.py file, we can also achieve various facial effects I guess.

**Comment by tjasmin111** [comment]
Yeah they may. But using a gan would be slow. Theoretically if we get a mesh and vertex numbers, we can just change their coordinates.
Good idea. I wish someone could start doing this. There is no resources online
